packages/flutter/lib/src/widgets/bottom_navigation_bar_item.dart

```diff
@@ -21,7 +21,7 @@ import 'framework.dart';
 class BottomNavigationBarItem {
   /// Creates an item that is used with [BottomNavigationBar.items].
   ///
-  /// The argument [icon] should not be null and the argument [title] should not be null when used in a Material Design's [BottomNavigationBar].
+  /// The argument [icon] should not be null and the argument [title] or [label] should not be null when used in a Material Design's [BottomNavigationBar].
```

Comment:
I think this comment should be updated to only mention `label`, since that is the non-deprecated parameter.

Reply:
Let me double check that on Monday

Reply:
Yeah, let's only mention the `label` here.

Reply:
Ok Thanks @goderbauer

Reply:
Thanks, @goderbauer